docs/how-to/walkthrough.md

```diff
@@ -0,0 +1,314 @@
+# Simple Text Search
+
+Let's build a Flow that will index text Documents into an index so that we can then retrieve them using Jina framework and run in Kubernetes.
+
+We will see:
+ - How to create a custom Executor
+ - How to publish privately to Jina Hub
+ - How to build your Flow locally combining different Executors
+ - How to deploy it in Kubernetes
+
+For that we want to build a Flow that looks like this:
+
+```{figure} simple_flow_walkthrough.svg
+:width: 70%
+
+```
+
+In this case, we are going to see how to build a Flow when we will use a public Executor from the `Hub` and a private Executor built and developed by us.
```

Comment:
I don't get this part, it's about creating a Flow to be used with public executors. And then, different topic, creating a private Executor?